chromium_src/ui/views/bubble/bubble_border.h

```diff
@@ -6,13 +6,6 @@
 #ifndef BRAVE_CHROMIUM_SRC_UI_VIEWS_BUBBLE_BUBBLE_BORDER_H_
 #define BRAVE_CHROMIUM_SRC_UI_VIEWS_BUBBLE_BUBBLE_BORDER_H_
 
-#include "build/build_config.h"
-
-#if defined(OS_WIN)
-// Prevent overriding global GetClientRect Win32 api.
-#include <Windows.h>
-#endif
-
```

Comment:
@simonhong, I couldn't see why this was needed (any more?).

Reply:
Windows build is fine with C94? I still can see below build error with current master(c94).
```
In file included from ../../brave/browser/ui/views/sidebar/bubble_border_with_arrow.cc:6:
In file included from ../..\brave/browser/ui/views/sidebar/bubble_border_with_arrow.h:9:
In file included from ../../brave/chromium_src\ui/views/bubble/bubble_border.h:17:
In file included from ../../brave/chromium_src\ui/views/bubble/../../../../../ui/views/bubble/bubble_border.h:15:
In file included from ../..\ui/views/background.h:21:
In file included from ../../../../../Program Files (x86)/Windows Kits/10/include/10.0.19041.0/um\windows.h:174:
../../../../../Program Files (x86)/Windows Kits/10/include/10.0.19041.0/um\winuser.h(8973,1): error: 'virtual' can only appear on non-static member functions
GetClientRect(
^
../../brave/chromium_src\ui/views/bubble/bubble_border.h(16,23): note: expanded from macro 'GetClientRect'
#define GetClientRect virtual GetClientRect
                      ^
1 error generated.
```

Reply:
Ok, I built cr95 branch on my local and didn't get above error.
Due to below code from `background.h`, we got compile error.
But it's deleted from cr95. This deletion seems fine for me. :) 
```
#if defined(OS_WIN)
#include <windows.h>
#endif  // defined(OS_WIN)
```

Reply:
Oh wow @simonhong thanks for checking! Didn't mean to put all that on you. I figured it was just cleanup of headers upstream but wanted to make sure it didn't have any additional meaning. Thanks again for checking 👍

Reply:
My pleasure!